File: Dynamic Programming Applications In Machine Learning and Genomics/Minimum Number of Coins to Make Change /ChangeCoins.java
import java.util.Scanner;

public class ChangeCoins {
    private static int findChange(int money, int[] coins) {
        // TODO: your code here
        return -1;
    }

    public static void main(String[] args) {
        Scanner stdin = new Scanner(System.in);
        int money = stdin.nextInt();
        stdin.nextLine();

        String[] rawCoins = stdin.nextLine().split(",");
        int[] coins = new int[rawCoins.length];

        for (int i = 0; i < rawCoins.length; i++) {
            coins[i] = Integer.parseInt(rawCoins[i]);
        }

        System.out.println(findChange(money, coins));
    }
}

File: Dynamic Programming Applications In Machine Learning and Genomics/Minimum Number of Coins to Make Change /change_coins.py
#python3
import sys

def find_change(money, coins):
    # TODO: your code here
    return -1

if __name__ == "__main__":
    money = int(sys.stdin.readline().strip())
    coins = list(map(int, sys.stdin.readline().strip().split(',')))
    print(find_change(money, coins))

